src/poliastro/earth/sensors.py

```diff
@@ -1,110 +1,107 @@
-import numpy as np
 from astropy import units as u
 
+from poliastro.core.sensors import (
+    max_and_min_ground_range_with_specific_azimuth as max_and_min_ground_range_with_specific_azimuth_fast,
+    min_and_max_ground_range as min_and_max_ground_range_fast,
+)
+
 
-@u.quantity_input(h=u.km, η_fov=u.rad, η_center=u.rad, R=u.km)
-def min_and_max_ground_range(h, η_fov, η_center, R):
+@u.quantity_input(h=u.km, n_fov=u.rad, n_center=u.rad, R=u.km)
```

Comment:
If we're doing this, let's not rename `η` to `n`, and use `eta` instead.

```suggestion
@u.quantity_input(h=u.km, eta_fov=u.rad, eta_center=u.rad, R=u.km)
```

(this applies everywhere)